app/controllers/miq_ae_class_controller.rb

```diff
@@ -2318,7 +2318,9 @@ def fields_get_form_vars
             fld[field] = params[var_name.to_sym] if params[var_name.to_sym]
           elsif field == "default_value"
             fld[field] = params[field_name] if params[field_name]
-            fld[field] = params["fields_password_value_#{i}".to_sym] if params["fields_password_value_#{i}".to_sym]
+            if fld['datatype'] == 'password'
+              fld[field] = params["fields_password_value_#{i}".to_sym] if params["fields_password_value_#{i}".to_sym]
+            end
```

Comment:
I don't understand this change?  Doesn't this just do what the field already does?

Reply:
After I add or delete a row, it sends a second `fields_form_field_changed` request with a payload with a password_field_value and the original value in default value even though the datatype isnt a password. 

So then when a deletion or addition happens afterwards, the value is set back to the original
<img width="693" alt="Screenshot 2024-10-22 at 4 06 34 PM" src="https://github.com/user-attachments/assets/2299d8eb-bf47-4314-ac2f-dfbbc2939347">